src/app/core/directives/server-html.directive.ts

```diff
@@ -21,6 +21,10 @@ import { LinkParser } from 'ish-core/utils/link-parser';
   selector: '[ishServerHtml]',
 })
 export class ServerHtmlDirective implements AfterContentInit, AfterViewInit, OnDestroy, OnChanges {
+  @Input() callbacks?: {
```

Comment:
I think, the optional mark `?` is not necesssary here since this property can remain `undefined`.